src/main/java/org/jenkinsci/plugins/ghprb/Ghprb.java

```diff
@@ -37,7 +37,7 @@
  * @author janinko
  */
 public class Ghprb {
-    private static final Logger logger = Logger.getLogger(Ghprb.class.getName());
+    private static final Logger logger = Logger.getLogger(Ghprb.class.getPackage().getName());
```

Comment:
i think these changes should be reverted. It's quite common to use the class name as a logger name.